app/gateway/2.8.x/plan-and-deploy/security/secrets-management/backends-overview.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Supported Vault Backends
+beta: true
+---
+
+{:.warning}
+> **Beta warning:** Secrets rotation is currently not supported in this version.
```

Comment:
```suggestion
Secrets rotation is not supported for the beta version.
```

Since this page is tagged as beta, this is happening now:

![Screen Shot 2022-02-22 at 11 08 57 AM](https://user-images.githubusercontent.com/54370747/155201800-5a7bee70-c95b-42c5-a9d7-9aa10725bf75.png)